### DISCUSSION TITLE
Automatic file reload

### DISCUSSION THREAD

**Discussion by s0la** [neovim/neovim#13609] [Q&A] [answered]
Hey guys,

After installing neovim from source about a week ago, I noticed neovim automatically reloads files which were changed..
I didn't see this behaviour before 5.0 version (before the upgrade), so I assume it is not caused by any of my config files, but rather that it's default behaviour in new version..

Did anyone else noticed the same behaviour, or is it actually my doing, and do you know how can I disable it?
Thanks in advance :)

**Comment by fsouza** [comment, marked as answer]
Yes, you can disable autoread with `set noautoread`. autoread has defaulted to `on` on neovim for a while, but as far I as understand, it was kinda broken and is now fixed on master / 0.5.

Other people may be able to provide more context, but `set noautoread` is what you want x)

**Reply by s0la** [reply]
Hey, thank you so much for such a fast response!! That sounds exactly like what I was looking for, thanks again, mate!
Cheers! :)

**Comment by teto** [comment]
our gsoc student added some code that checks your buffers on focusLost/FocusGained events which "improve" autoread. The real fix will be after 0.5 with https://github.com/neovim/neovim/pull/12593

**Reply by s0la** [reply]
Great to hear that, thanks!! :)
